source/develop/release-management/features/storage/cd-block-storage.md

```diff
@@ -0,0 +1,228 @@
+---
+title: CD-ROM support for block storage
+category: feature
+authors: vjuranek, nsoffer
+feature_name: CD-ROM support for block storage
+feature_modules: engine,vdsm
+feature_status: Design
+---
+
+# CD-ROM support for block storage
+
+
+## Summary
+
+This feature adds new functions for CD-ROM management, namely for loading, ejecting and changing CD-ROM.
+
+## Owner
+
+- Vojtech Juranek (<vjuranek@redhat.com>)
+
+## Overview
+
+### What was available before this feature?
+
+Before this, feature oVirt engine provided functions for changing and ejecting CD-ROM.
+However, non of these functions worked properly with block based storage.
+Changing CD failed on block storage as the appropriate volume wasn't activated.
+Ejecting CD failed to deactivate the volume.
+For more details, see following bugs.
```

Comment:
I think the guest OS will also be more happy if there is no CD change without "opening the tray" anymore.

Reply:
What do you mean?

Reply:
If a CD change is implemented as eject + insert then it is what the guest OS is used to. If a CD is changed in-place, without ejecting the old CD first, it can confuse the guest OS and lead to various problems. We had such issues in the past.